src/main/markdown/doc/latest/tutorial/RPC.md

```diff
@@ -1,5 +1,5 @@
 Using GWT RPC
```

Comment:
I'd suggest removing == and put ## or for the very first, use one # for the title.